helix-view/src/editor.rs

```diff
@@ -459,6 +459,8 @@ pub struct LspConfig {
     pub display_signature_help_docs: bool,
     /// Display inlay hints
     pub display_inlay_hints: bool,
+    /// Max length of inlay hints, if it's none, there's no limit
+    pub inlay_hints_length_limit: Option<u8>,
```

Comment:
You can use a `NonZeroU8` here to make it impossible to configure this as `0`. (And it saves a byte of space.)